internal/mode/static/nginx/config/servers_test.go

```diff
@@ -972,8 +964,10 @@ func TestCreateLocationsRootPath(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		locs := createLocations(test.pathRules, 80)
-		g.Expect(locs).To(Equal(test.expLocations), fmt.Sprintf("test case: %s", test.name))
+		t.Run(test.name, func(t *testing.T) {
+			locs := createLocations(test.pathRules, 80)
+			g.Expect(locs).To(Equal(test.expLocations), fmt.Sprintf("test case: %s", test.name))
```

Comment:
You don't need the additional context here anymore since the subtest framework will create a test with the `test.name`. 
```suggestion
			g.Expect(locs).To(Equal(test.expLocations))
```

Reply:
Should I leave the additional context on the test that isn't using the subtest framework (in the secret_test.go file)

Reply:
Yep!